### PR TITLE
Add tool tip for shape lock

### PR DESCRIPTION
**Pull request by lipis**

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -61,6 +61,7 @@
     "ellipse": "Ellipse",
     "arrow": "Arrow",
     "line": "Line",
-    "text": "Text"
+    "text": "Text",
+    "lock": "Lock the currently selected element"
```
Comment (lipis) [inline public/locales/en/translation.json]: open for ideas..

Comment (dwelle) [reply public/locales/en/translation.json]: I think that suggests it'll lock the element in place (the way you lock layers in editors). I'd use something like "Keep selected tool active". Dunno, doesn't sound good, either.

Comment (voluntadpear) [reply public/locales/en/translation.json]: "Keep selected tool active after drawing"?

**Comment by dwelle** [comment]
Will make the change. Should we wait for translations?

**Comment by voluntadpear** [comment]
> Will make the change. Should we wait for translations?

I think it's hard to coordinate translations for all languages at this point. It shouldn't be a blocker for merging PRs. Missing translations can be added at the moment native speakers check what they can add.

**Comment by dwelle** [comment]
Right. Yea, I agree. We should later add some CLI utility to list what translations are missing for what language, to make that easier.

**Comment by dwelle** [comment]
Merging :rocket:

**Pull request merged**